src/leap/mx/alias_resolver.py

```diff
@@ -64,3 +74,7 @@ class AliasResolverFactory(LEAPPostfixTCPMapServerFactory):
     """
 
     protocol = LEAPPostfixTCPMapAliasServer
+
+    @property
+    def _query_message(self):
+        return "Resolving alias for"
```

Comment:
The same class, in the `get()` method right below. The idea is to allow per-subclass error messages, in a way we don't get "Query key: name@domain" messages in the log. The "key" may be a pgp public key or a query key in tcp map's terminology.